vscode/tslint.json

```diff
@@ -0,0 +1,15 @@
+{
+	"rules": {
```

Comment:
It seems like the code contains tabs as well as spaces. Should we add a corresponding [rule](https://palantir.github.io/tslint/rules/indent/) and fixer?

Reply:
It seems like the tslint check is currently not integrated in the package.json as script.  Do you use tslint? We could feed "codacy" with the tslint.json in order to check for violations.

Reply:
@Drakulix mentioned that [tslint is deprecated](https://github.com/palantir/tslint/issues/4534).

Reply:
Introduced eslint as suggested, base rules are the google ones. Might be worth considering using the more popular ones from airbnb later. Can't really tell which one is better but since java also uses googles recommendations I used those here too.